Keybinds.cpp

```diff
@@ -178,31 +178,31 @@ action_name_mapper actionnamemap[] = {
     {"ROTATE", action_incrrotation},
     {"RELOAD_SEGMENT", action_reloadsegment},
     {"PAINT", action_paintboard},
-    {"TOGGLE_DESIGNATIONS", action_toggledesignations},
-    {"TOGGLE_STOCKS", action_togglestockpiles},
-    {"TOGGLE_ZONES", action_togglezones},
+    //{"TOGGLE_DESIGNATIONS", action_toggledesignations},
```

Comment:
if these are going away, just delete them, don't comment them out
the old code will always be available from the commit history